test/ctia/http/routes/graphql/investigation_test.clj

```diff
@@ -0,0 +1,84 @@
+(ns ctia.http.routes.graphql.investigation-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.test :refer [deftest is join-fixtures testing use-fixtures]]
+            [ctia.entity.investigation :as inv]
+            [ctia.entity.investigation.examples :refer [investigation-maximal]]
+            [ctia.test-helpers.auth :refer [all-capabilities]]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.graphql :as gh]
+            [ctia.test-helpers.store :refer [test-for-each-store]]))
+
+(use-fixtures :once (join-fixtures [mth/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(def ownership-data-fixture
+  {:owner "foouser"
+   :groups ["foogroup"]})
+
+(def investigation-1 (fixt/randomize investigation-maximal))
+(def investigation-2 (assoc
+                      (fixt/randomize investigation-maximal)
+                      :source "ngfw"))
+
+(defn select-graphql-fields
+  [investigation]
+  (select-keys investigation
+               [:id
+                :type
+                :schema_version
+                :title
+                :timestamp
+                :tlp
+                :source
+                :owner
+                :groups
+                :object_ids
+                :investigated_observables
+                :targets]))
+
+(deftest investigation-graphql-test
+  (test-for-each-store
+   (fn []
+     (helpers/set-capabilities! "foouser" ["foogroup"] "user" all-capabilities)
+     (whoami-helpers/set-whoami-response "45c1f5e3f05d0"
+                                         "foouser"
+                                         "foogroup"
+                                         "user")
+     (let [inv1 (select-graphql-fields
+                 (gh/create-object "investigation" investigation-1))
+           inv2 (select-graphql-fields
+                 (gh/create-object "investigation" investigation-2))
+           graphql-queries (slurp "test/data/queries.graphql")]
+       (testing "investigations query"
```

Comment:
Could you also test the `investigation` Graphql query? This part has not been moved in this ns.